pkg/build/builder/daemonless.go

```diff
@@ -193,7 +193,9 @@ func pullDaemonlessImage(sc types.SystemContext, store storage.Store, imageName
 		SystemContext: &systemContext,
 		BlobDirectory: blobCacheDirectory,
 	}
+	log.V(0).Infof("Calling buildah to pull %s", imageName)
 	_, err = buildah.Pull(context.TODO(), "docker://"+imageName, options)
+	log.V(0).Infof("Returned from calling buildah to pull %s with error received %v", imageName, err != nil)
```

Comment:
I don't think we need to log the error here - if one is raised, we'll log it elsewhere.

```suggestion
	log.V(0).Infof("Finished pulling image %s", imageName)
```